models/dex/dex_trades_migration_beta.sql

```diff
@@ -69,6 +69,7 @@
 ,ref('zipswap_trades')
 ,ref('balancer_trades')
 ,ref('hashflow_trades')
+,ref('honeyswap_v2_trades')
```

Comment:
for clarity, this would also be updated to new file name `honeyswap_trades`

Reply:
@jeff-dude I'd be okay with doing that - but the reason I did it this way is to keep the same name as the decoded tables within Dune

Reply:
finally figured it out in https://github.com/duneanalytics/spellbook/pull/4139/commits/9405b75f5a9facc1f1f3d0ce30e5cc28bcbf311f